client/src/js/services/DepotService.js

```diff
@@ -29,16 +32,37 @@ function DepotService($http) {
   // GET a list of depots for current enterprise.  If an ID is supplied, returns
   // the details of a single depot.
   function getDepots(depotId) {
-    return get(depotId === undefined ? '/depots' :'/depots/' + depotId);
+    var url = baseUrl.concat(depotId || '');
+    return get(url);
```

Comment:
I know that you didn't write this code, but might as well update it while you are here:
```js
var url = baseUrl.concat(depotId || '');
return $http.get(url)
  .then(util.unwrapHttpResponse);
```
... and then you can delete the duplicate `.get()` method.